### DISCUSSION TITLE
about block component, how to custom simplified head & body like vitepress

### DISCUSSION THREAD

**Discussion by leosin** [nuxt/content#1932] [General]
i have some question with block component.

When using vitepress before, the text next to the tip can be used as the title
like this:
![image](https://user-images.githubusercontent.com/3676670/221480364-03f6652b-29f1-4716-84ef-d75a00146b74.png)

in content, we need to add `#head` and `body`
```md
:: tip
#hade
The TItle

#body
some text....
::
```
so ,is there any way to use `:: tip The title` directly.

because i am migrating a lot of documents from vitepress

**Comment by ManasMadrecha** [comment]
Using your own custom `remark` plugins, you can have any syntax you want. 

EDIT: Here is what you are looking for https://github.com/ipikuka/remark-flexible-containers

**Reply by leosin** [reply]
Thank you very much, this looks good. 👍
